manifests/m/MarekJasinski/FreeCommanderXE/2023.0.0.880/MarekJasinski.FreeCommanderXE.locale.de.yaml

```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.4.0.schema.json
+
+PackageIdentifier: MarekJasinski.FreeCommanderXE
+PackageVersion: 2023.0.0.880
+PackageLocale: de
+Publisher: Marek Jasinski
+PublisherUrl: https://freecommander.com/de/ubersicht/
+PublisherSupportUrl: https://freecommander.com/fchelpxe/de/FreeCommander.html
+PrivacyUrl: https://freecommander.com/de/datenschutz/
+Author: Marek Jasinski
+PackageName: FreeCommander XE
+PackageUrl: https://freecommander.com/de/ubersicht/
+License: Proprietary
+LicenseUrl: https://freecommander.com/de/lizenz/
+Copyright: Copyright © 2004-2023 Marek Jasinski. All rights reserved
+ShortDescription: Total Commander ist ein Filemanager im Stil des Norton Commanders.

```

Comment:
```suggestion
ShortDescription: FreeCommander XE ist eine leicht bedienbare Alternative zum Standard-Dateimanager.
```

Reply:
Hi @borque,

Same situation as above, and also the product name is slightly different. I'm not very familiar with German, only working with internet translations, so feel free to update this if needed.